deployment/gcp/sql.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  db_admins = [
```

Comment:
Should this be a IAM GCP Group? Or are we good with individual admins for now?

Reply:
GCP "groups" require a Google Worklspace. I wouldn't normally have the breakglass account either, but it seems prudent. Should I remove those "admin" individual users?